SIL/script.js

```diff
@@ -0,0 +1,85 @@
+window.addEventListener("load", function(){
```

Comment:
Another code style tip: we developers like to keep our quotes consistent. In Javascript, we usually go for single quotes (`'string'`). Basically, I suggest that you choose a quote style for your file or project and keep it along the way.

One exception to the single quote rule is React. When writing `jsx`, we usually use double quotes for attribute names (`<div className="header">`).